lib/_http_server.js

```diff
@@ -447,6 +450,8 @@ function storeHTTPOptions(options) {
   this[kIncomingMessage] = options.IncomingMessage || IncomingMessage;
   this[kServerResponse] = options.ServerResponse || ServerResponse;
 
+  this[kDumpGet] = Boolean(options.dumpGETBody ?? globalThis.IS_NODE_TEST);
```

Comment:
I guess this new option in `global` should be documented somehow

Reply:
instead of using the `globalThis` (which afaik should not be done) you could expose a internal symbol (when --expose-internals is enabled) that allows you to change the value of a property in the http server